calblueprint/ks-web
# myles/files_update

[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?
1. Deleted and/or renamed (most) PP-Solar folders, components, and styles
2. Merged adminUtils and onboardingUtils => utils
3. Added path aliases for common folders
4. Added `route` folder for routing-related components
5. Removed 'Forgot Password' and 'Create Account' to make way for updated designs

### How to Review
1. Pull branch and start development
2. Ensure that components look similar to master.
3. Check that path aliases are working correctly (there should be no import console errors)

### Related PRs
[//]: # "Optional - related PRs you're waiting on
/ PRs that will conflict, etc"
1. Merge ASAP, future PRs will likely conflict
### Next steps

[//]: # "What doesn't work yet, what's NOT in this 
PR/has to be done "
1. Add React.Suspense and Lazy Loading for Routes
2. Reorganize file directory structure
3. Replace onboarding => sign up forms
4. @spring to create 'Forgot Password' interactions and 'Create Account' form designs
### Screenshots
![image](https://user-images.githubusercontent.com/35586281/110434787-19563900-8067-11eb-9a9a-761e394d6966.png)
![image](https://user-images.githubusercontent.com/35586281/110434821-2410ce00-8067-11eb-8b29-0ca4e4e5c6f7.png)
![image](https://user-images.githubusercontent.com/35586281/110434849-2b37dc00-8067-11eb-8133-3ee1205442d3.png)
![image](https://user-images.githubusercontent.com/35586281/110434873-312dbd00-8067-11eb-9388-31e576e96cd4.png)
![image](https://user-images.githubusercontent.com/35586281/110434892-37239e00-8067-11eb-9234-2ae34e5cff9f.png)
![image](https://user-images.githubusercontent.com/35586281/110434918-3d197f00-8067-11eb-884c-fada31e09d5d.png)
![image](https://user-images.githubusercontent.com/35586281/110435440-e19bc100-8067-11eb-937a-0846a2439df4.png)

^ above is a *rough* estimate of modified files



CC: @phoebeli23
